src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -20,7 +20,6 @@
     <PackageReference Include="protobuf-net" Version="2.3.7" />
     <PackageReference Include="System.Drawing.Common" Version="4.5.1" />
     <PackageReference Include="System.IO.Pipelines" Version="4.5.3" />
-    <PackageReference Include="System.Memory" Version="4.5.1" />
```

Comment:
Didn't seem to be required. I was able to build and run tests without this. 

for example before even adding new tests (which includes the new FirstSpan API), I wouldn't have needed the local reference below on line 40 and also without this packagereference to System.Memory I could still get the benchmarks running

Reply:
After removing this package reference, does it still build/run in all target frameworks?

Reply:
Good question. Let me check...

Reply:
Yes, without PackageReference to System.Memory it still builds in the following target frameworks:

- net461
- netcoreapp2.0
- netcoreapp2.1
- netcoreapp2.2
- netcoreapp3.0